react/src/components/Orcid.js

```diff
@@ -0,0 +1,70 @@
+
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import Glyphicon from 'react-bootstrap/lib/Glyphicon';
+
+import EduIDButton from 'components/EduIDButton';
+
+import 'style/AccountLinking.scss';
+
+
+class Orcid extends Component {
+
+  render () {
+    let spinning = false;
+    let orcidData;
+
+    if (this.props.is_fetching) spinning = true;
+
+    const orcidIntro = (
+        <div className="orcid-intro">
+          <h4>{this.props.l10n('orc.title')}</h4>
+          <p>{this.props.l10n('orc.long_description')}</p>
+        </div>
+    );
+
+    if (this.props.orcid.id !== undefined) {
```

Comment:
With no orcid data, the value of my state.account_linking.orcid is null, which has no attribute id